src/frontend/dash/busBubble.ts

```diff
@@ -0,0 +1,80 @@
+import { Circle, ClockCircle, getBounds } from "./circlePacker"
+import { MemberCircle } from "./circlePacker"
+import {getArrivals, busSigns} from "./trimet"
+
+
+function renderClock(circle: ClockCircle, alone?:boolean) {
+    
+//     <div class="minutes">
+// </div>
+    elem.style.width = elem.style.height = radius + "vw";
+    elem.style.left = (circle.position.x - minX) * multiplier + offsetX - radius/2 + "vw";
+    elem.style.top = (circle.position.y - minY) * multiplier + offsetY - radius/2 + "vw";
+    elem.style.fontSize = Math.min(radius) + "vw"
+
+    if(alone){elem.classList.add('clockCircleAlone')}
+
+    // elem.style.backgroundImage = `url(${circle.imgurl})`
+    // elem.innerHTML = 'hi'
```

Comment:
Are these styles that you'll want to switch back and forth from frequently? If not, it might be best to remove them for clarity, or mark them as debugging code.

Reply:
I'll remove them :3